packages/app/non-translatable-markup/src/methods/isTextTranslatable.ts

```diff
@@ -1,10 +1,16 @@
-import { nonTranslatableTextRegexp } from './utils'
+import { dateRegexp, nonTranslatableTextRegexp, symbolsAndNumberRegexp } from './utils'
 
 /**
- * Returns true if the text is entirely encapsulated in non-translatable tags.
+ * Returns true if the text contain at least one translatable piece of text.
+ * A translatable piece of text is a piece of text that is not surrounded by non-translatable tags,
+ * symbols or a number.
  */
-export const isTextTranslatable = (text: string): boolean => {
-  const parts = text.split(nonTranslatableTextRegexp).map((part) => part.trim())
+export const isTextTranslatable = (text: string): boolean =>
+  text
+    .split(nonTranslatableTextRegexp)
+    .map((p) => p.trim())
```

Comment:
it's not clear what `p` stands for, can we make it more explicit?

Reply:
Good point! already done, thank youuu